crates/fuel-core/src/graphql_api/worker_service/tests.rs

```diff
@@ -81,6 +81,7 @@ fn worker_task_with_block_importer_and_db<D: ports::worker::OffChainDatabase>(
         block_importer,
         database,
         chain_id,
+        da_compression_config: DaCompressionConfig::Disabled,
```

Comment:
It would be nice to have a unit test for it later to be sure that we process the block when it is enabled. Can be a good first(for new team members) follow-up PR=)

Reply:
We do have an integration test here https://github.com/FuelLabs/fuel-core/blob/dc34257c3263639f1a023ea337b464f192487fde/tests/tests/da_compression.rs#L30

but maybe we could have an unit test as well, yes.